.coveragerc

```diff
@@ -1,3 +1,7 @@
 [run]
 branch = true
 include = w3lib/*
+
+[report]
+exclude_lines =
+    if TYPE_CHECKING:
```

Comment:
This may be a good chance to move this config to `pyproject.toml`, [it seems to be supported](https://coverage.readthedocs.io/en/latest/config.html).